modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -3,7 +3,7 @@
 You can enable the Bulk Import feature for users and give them the necessary permissions to access it.
 
 .Prerequisites
-* You have link:{authentication-book-url}#enabling-authentication-with-github[configured GitHub authentication and integration].
+* You have link:{authentication-book-url}#enabling-authentication-with-github[configured GitHub integration].
```

Comment:
Thanks for the prompt fix. There is another reference to GitHub Authentication in "Importing multiple GitHub repositories": https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-646/getting-started-rhdh/#importing-multiple-github-repositories